CONTRIBUTING.md

```diff
@@ -27,6 +47,67 @@ pull request reviewed, the reviewer(s) may ask you to complete additional
 work, tests, or other changes before your pull request can be ultimately
 accepted.
 
+# For maintainers
```

Comment:
```suggestion
# Developing for Dart Frog
```

In Very Good CLI and other contributing files we name the section "Developing for X". It was inspired by Flutter's CONTRIBTUING, see [this](https://github.com/flutter/flutter/blob/master/CONTRIBUTING.md#developing-for-flutter).